diff --git a/examples/python/maintenance/__init__.py b/examples/python/maintenance/__init__.py
index c39f3a9..3e1c9fb 100644
--- a/examples/python/maintenance/__init__.py
+++ b/examples/python/maintenance/__init__.py
@@ -25,7 +25,7 @@ class Entity:
             timestamp = datetime.now().strftime('%y%m%d')
             standardized = f'A_{name}_{timestamp}'
         elif self.standard_ == 'B':
-            timestamp = datetime.now().strftime('%Y%m%d')
+            timestamp = datetime.now().strftime('%y%m%d')
             standardized = f'B{name}{timestamp}'
 
         self.tasks_.append(Task(standardized, priority))
