diff --git a/examples/c/src/maintenance.c b/examples/c/src/maintenance.c
index a645878..14c1365 100644
--- a/examples/c/src/maintenance.c
+++ b/examples/c/src/maintenance.c
@@ -73,7 +73,7 @@ void entity_add_task(struct entity *self, const char *name, int priority)
         strftime(timestamp, 10, "%y%m%d", localtime(&now));
         sprintf(standardized, "A_%s_%s", name, timestamp);
     } else if (self->standard == B) {
-        strftime(timestamp, 10, "%Y%m%d", localtime(&now));
+        strftime(timestamp, 10, "%y%m%d", localtime(&now));
         sprintf(standardized, "B%s%s", name, timestamp);
     }
 
