diff --git a/examples/c/src/maintenance.c b/examples/c/src/maintenance.c
index a645878..7e3e9c8 100644
--- a/examples/c/src/maintenance.c
+++ b/examples/c/src/maintenance.c
@@ -115,7 +115,7 @@ bool_t entity_should_process(struct entity *self)
 
 void entity_prioritized_tasks(struct entity *self, struct tasks_vector *output)
 {
-    if (self->waiting >= self->waiting_threshold) {
+    if (self->waiting >= self->tasks_threshold) {
         self->waiting = 0;
         tasks_vector_extend(output, self->tasks);
     } else {
